src/main/scala/com/microsoft/hyperspace/index/IndexConstants.scala

```diff
@@ -57,4 +57,7 @@ object IndexConstants {
 
   val REFRESH_DELETE_ENABLED = "spark.hyperspace.index.refresh.delete.enabled"
   val REFRESH_DELETE_ENABLED_DEFAULT = "false"
+
+  val ENFORCE_DELETE_ON_READ_ENABLED = "spark.hyperspace.index.enforce.delete.on.read.enabled"
```

Comment:
btw, you don't have to have "." for each word. you can do "deleteOnRead.enabled", for example.